src/Analyzer/ValidationUtils.cpp

```diff
@@ -276,6 +276,9 @@ void validateAggregates(const QueryTreeNodePtr & query_node, AggregatesValidatio
         if (query_node_typed.hasOrderBy())
             validate_group_by_columns_visitor.visit(query_node_typed.getOrderByNode());
 
+        if (query_node_typed.hasInterpolate())
+            validate_group_by_columns_visitor.visit(query_node_typed.getInterpolate());
```

Comment:
@novikd I suspect it's not correct - for `INTERPOLATE (inter AS mn * 2)` - `getInterpolate()` returns left side (`inter`) - it's just a placeholder - the calculated expression is the right side (`mn * 2`) - returned by `getInterpolateExpression()`
```suggestion
            validate_group_by_columns_visitor.visit(query_node_typed.getInterpolateExpression());
```

Reply:
No, my change is correct because here `QueryNode::getInterpolate()` is called and this method returns `InterpolateNode`. As you can see, the test passed.

You confused it with `InterpolateNode::getExpression()`.

Reply:
oh, right, I confused it...